tck/src/main/java/ee/jakarta/tck/data/framework/read/only/AsciiCharacter.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+package ee.jakarta.tck.data.framework.read.only;
+
+import java.io.Serializable;
+
+@jakarta.nosql.Entity
+@jakarta.persistence.Entity
+public class AsciiCharacter implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    @jakarta.nosql.Id
+    @jakarta.persistence.Id
+    private long id;
+
+    @jakarta.nosql.Column
+    private int decimal;
```

Comment:
`DECIMAL` is a reserved word in some databases.  Maybe rename the field to `numericValue` or use the annotations to give it a different name in the database?

Reply:
Good catch, fixed 👍🏼